### PR TITLE
homework #8

### PR DESCRIPTION
**Pull request by AndryNosach**
servlets changed to controllers

### CODE REVIEW

File: Nosach/web-quiz-project/src/main/java/controllers/QuizController.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+    @RequestMapping(method = POST, value = "add")
+    public String addQuizBasicInfo(HttpSession session, WebRequest req){
+        if(session.getAttribute("builder") == null){
+            quizBuilder.clean();
+            session.setAttribute("builder", quizBuilder);
+        }
+
+        String subject;
+        if (req.getParameter("subjectList") != ""){
```
Comment (dinamio) [inline Nosach/web-quiz-project/src/main/java/controllers/QuizController.java:62]: Maybe it's better, to pass request parameter as parameter in method?

File: Nosach/web-quiz-project/src/main/java/controllers/UserController.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    public String getRegistrationPage() {
+        return "register-page";
+    }
+
+    @RequestMapping(method = POST, value = "register")
+    public String doRegistration(WebRequest req, HttpSession session) {
+        String login = req.getParameter("login");
+        String pass = req.getParameter("pass");
+        String email = req.getParameter("email");
+        String name = req.getParameter("name");
```
Comment (dinamio) [inline Nosach/web-quiz-project/src/main/java/controllers/UserController.java]: Better use spring forms

**Review by dinamio — APPROVED** [review]
Looks good

File: Nosach/web-quiz-project/src/main/java/controllers/QuizController.java (hunk 1 above)
Comment (AndryNosach) [reply Nosach/web-quiz-project/src/main/java/controllers/QuizController.java:62]: In this method I used three different request parametrs, so I desided not to separate them.

File: Nosach/web-quiz-project/src/main/java/controllers/UserController.java (hunk 2 above)
Comment (AndryNosach) [reply Nosach/web-quiz-project/src/main/java/controllers/UserController.java]: Agree. Changed code with using ModelAttribute.

**Pull request merged**